src/WebJobs.Extensions.DurableTask/PurgeHistoryResult.cs

```diff
@@ -15,30 +15,15 @@ public class PurgeHistoryResult
         /// <summary>
         /// Constructor for purge history statistics.
         /// </summary>
-        /// <param name="storageRequests">Requests sent to storage.</param>
         /// <param name="instancesDeleted">Number of instances deleted.</param>
-        /// <param name="rowsDeleted">Number of rows deleted.</param>
-        public PurgeHistoryResult(int storageRequests, int instancesDeleted, int rowsDeleted)
+        public PurgeHistoryResult(int instancesDeleted)
         {
-            this.purgeHistoryResult = new DurableTaskAzureStorage.PurgeHistoryResult(
-                storageRequests,
-                instancesDeleted,
-                rowsDeleted);
+            this.purgeHistoryResult = new DurableTaskAzureStorage.PurgeHistoryResult(0, instancesDeleted, 0);
```

Comment:
With this new implementation, I think you don't even need to include `DurableTaskAzureStorage.PurgeHistoryResult` anymore. Let's just store the int value to the `StorageRequests` property defined in your new class directly.